generate-web/README.md

```diff
@@ -0,0 +1,3 @@
+# Quicksart page
+
+This folder will be continuously delivered from develop branch to https://quickstart.holo.host/
```

Comment:
This couples how we're going to deploy this software, with, well, software. `generate-web` artifact should be self-sufficient and user should be able to download assets and run it on their own machine.

Reply:
It can be both self sufficient and deployed to a web address :-)